### ISSUE TITLE
Printing of log messages is limited to 1000 characters

### ISSUE THREAD

**Issue by pepriv** [sassoftware/python-swat#58]
This code can be used to reproduce the issue:

```
code = "str = ''; do i = 1 to 1000; str = str || 'foo'; end; print str;"
s.loadactionset('sccasl')
result = s.runcasl(code)
```

The results, which get printed automatically, end with `... truncated`

This only happens when using the binary interface, not REST. A workaround is to disable automatic printing and explicitly print the log messages like this:

```
with swat.option_context(print_messages=False):
    result = s.runcasl(code)
     for message in result.messages:
         print(message)
```

**Comment by kesmit13** [comment]
This has been fixed in the SWAT C extension internally.  It will be packaged with SWAT at the next CAS server release.

**Comment by kesmit13** [comment]
Fixed in v1.8.0
